src/components/Badge/Badge.style.ts

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components/native';
+
+import {
+  Theme,
+  getSize,
+  getRadiusBySize,
+  getAvatarBySize,
+  getColorByName,
+} from '../../common/themeSelectors';
+
+import { BadgeColors, BadgeTypes } from './Badge';
+
+
+interface ContainerProps {
+  color: BadgeColors;
+  theme: Theme;
+  type: BadgeTypes;
+}
+
+interface ContentProps {
+  color: BadgeColors;
+}
```

Comment:
acho que aqui também é bom fazer um pick da interface do badge, pra ficar bem amarradinho os tipos

```suggestion
type ContainerProps = Required<Pick<BadgeProps, 'color' | 'theme' | 'type'>>;
type ContentProps = Required<Pick<BadgeProps, 'color' | 'theme'>>;
```